lib/views/home/home_view.dart

```diff
@@ -145,6 +145,20 @@ class _HomeViewState extends State<HomeView> {
     );
   }
 
+  String topic(var topic) {
+    if (topic.bolt11 == null) {
+      return "bolt12";
+    }
+    return "bolt11";
+  }
+
+  String value(var value) {
+    if (value.bolt11 == null) {
+      return value.bolt12.toString();
+    }
+    return value.bolt11;
```

Comment:
Very nice! I had only a comment, the parameter of a function must always have a type! otherwise the reader do not know what kind of type it is
```suggestion
  String topic(Type topic) {
    if (topic.bolt11 == null) {
      return "bolt12";
    }
    return "bolt11";
  }

  String value(Type value) {
    if (value.bolt11 == null) {
      return value.bolt12.toString();
    }
    return value.bolt11;
```